RELEASE/scripts/autoscend/paths/avatar_of_sneaky_pete.ash

```diff
@@ -74,205 +74,213 @@ void pete_buySkills()
 	{
 		return;
 	}
-	//if you have those 3 skills then you have all skills.
-	if(have_skill($skill[Natural Dancer]) && have_skill($skill[Flash Headlight]) && have_skill($skill[Walk Away From Explosion]))
-	{
-		return;
-	}
 
-	string page = visit_url("da.php?place=gate3");
-	matcher my_skillPoints = create_matcher("<b>(\\d\+)</b> skill point", page);
-	if(my_skillPoints.find())
+	//if you have those 3 skills then you have all skills.
+	if (!have_skill($skill[Natural Dancer]) || !have_skill($skill[Flash Headlight]) || !have_skill($skill[Walk Away From Explosion]))
```

Comment:
why this change?
we prefer to do
"if X return"
instead of
"if !X {entire function goes here}"

Reply:
I did this because if you enter this function and half all 30 skills the function exits and won't upgrade the motorcycle.

Reply:
There's no reason to change this to `_auto_peteSkills`, there's no reason to check again after rollover if you haven't leveled up.

Reply:
And I didn't want to add another property for the motorcycle.